src/analyzer/analyzerebur128mit.cpp

```diff
@@ -0,0 +1,93 @@
+#include <analyzer/analyzerebur128mit.h>
+#include <QtDebug>
+
+#include "trackinfoobject.h"
+#include "util/math.h"
+#include "util/sample.h"
+#include "util/timer.h"
+
+static const float kReplayGain2ReferenceLUFS = -18;
+
+AnalyzerEbur128Mit::AnalyzerEbur128Mit(UserSettingsPointer config)
+        : m_pConfig(config),
+          m_initalized(false),
+          m_iBufferSize(0),
+          m_pState(nullptr) {
+}
+
+AnalyzerEbur128Mit::~AnalyzerEbur128Mit() {
+}
+
+bool AnalyzerEbur128Mit::initialize(TrackPointer tio,
+        int sampleRate, int totalSamples) {
+    if (loadStored(tio) || totalSamples == 0) {
+        return false;
+    }
+
+    m_pState = ebur128_init(2u,
+            static_cast<unsigned int>(sampleRate),
```

Comment:
Since we cast here from signed to unsigned, I would like to keep it.
... and make code analysis tools happy,